### ISSUE TITLE
Cannot save tab order

### ISSUE THREAD

**Issue by synthead** [jneilliii/OctoPrint-TabOrder#58]
Duplicate of https://github.com/jneilliii/OctoPrint-TabOrder/issues/54 (I cannot reopen issue).

These are my tab order settings:

![Image](https://github.com/user-attachments/assets/f113e1a4-3954-45d4-9440-edc1e81fb127)

After saving the settings, I see the tabs like so:

![Image](https://github.com/user-attachments/assets/aef31bde-15b4-449e-b715-0d99741d29ec)

This persists even after Ctrl+Shift+R.

Using OctoPrint 1.10.3, Tab Order 0.5.12.

**Comment by jneilliii** [comment]
What about if you press shift+f5?
